packages/cosmo-dycore/package.py

```diff
@@ -170,14 +174,16 @@ def cmake_args(self):
     @run_after('install')
     @on_package_attributes(run_tests=True)
     def test(self):
+        cwd = os.path.join(self.root_cmakelists_dir, 'test/tools')
         if '+build_tests' in self.spec:
+            with open(os.path.join(cwd, 'spec.yaml'), mode='w') as f:
+                f.write(self.spec.to_yaml())
             try:
                 subprocess.run([
-                    './test_dycore.py', '-s',
-                    str(self.spec), '-b',
+                    './test_dycore.py', '-s', 'spec.yaml', '-b',
```

Comment:
Isn't this an API breaking change, requiring a change in `test_dycore.py` in all branches/versions and breaking compatibility to `cosmo-dycore@5.09a.mch1.2.p2`?

Reply:
👍